tools/build_chain.sh

```diff
@@ -897,7 +898,7 @@ if [ -z ${docker_mode} ];then
         echo "Binary check passed."
     fi
 fi
-
+latest_version=$(${bin_path} --version | grep "FISCO-BCOS Version" | cut -d':' -f2 | sed s/[[:space:]]//g)
```

Comment:
make use of `version=$(curl -s https://raw.githubusercontent.com/FISCO-BCOS/FISCO-BCOS/master/release_note.txt | sed "s/^[vV]//")`.
change to `latest_version =$(curl -s https://raw.githubusercontent.com/FISCO-BCOS/FISCO-BCOS/master/release_note.txt | sed "s/^[vV]//")`
make `latest_version` global

Reply:
`latest_version` is global. Pls modify this shell script according to your opinion. And modify the wiki.